docs/7-infrastructure-configuration-management/7.1.3-terraform-modules.md

```diff
@@ -145,13 +145,13 @@ module "ec2_instance" {
 
 2. Create a new `.tf` file or add the following configuration to your `main.tf` file from previous sections of the boot camp. This is declaring the module that we will use. The next steps we lay out the steps for creating this local module.
 
+?> Replace the `bucket_name` with an override to the local modules default
+
 ```hcl
 module "website_s3_bucket" {
   source = "./modules/s3-bucket"
 
-  bucket_name = var.bucket_name
-  tags = var.tags
-
+  bucket_name = "<bucket-override-name>"
```

Comment:
The before example was telling you to add this hcl section to your outer main.tf which did not have access to the child modules variables.tf file (unless I am doing something wrong). This was resulting in a variable not declared error before